controllers/resources/bentorequest_controller.go

```diff
@@ -1103,32 +1103,44 @@ func (r *BentoRequestReconciler) generateImageBuilderPod(ctx context.Context, op
 	bentoDownloadURL := opt.BentoRequest.Spec.DownloadURL
 	bentoDownloadHeader := ""
 
-	if bentoDownloadURL == "" {
-		var yataiClient_ **yataiclient.YataiClient
-		var yataiConf_ **commonconfig.YataiConfig
+	var yataiClient_ **yataiclient.YataiClient
+	var yataiConf_ **commonconfig.YataiConfig
 
-		yataiClient_, yataiConf_, err = getYataiClient(ctx)
-		if err != nil {
-			err = errors.Wrap(err, "get yatai client")
-			return
-		}
+	yataiClient_, yataiConf_, err = getYataiClient(ctx)
```

Comment:
We should only fetch the yatai client when bentoDownloadURL is empty. This is because we want to support yatai-image-builder running in clusters without yatai/bentocloud. In such cases, users can only use bentoDownloadURL to specify the download address for bento.

Reply:
yes